docs/mmteb/points.md

```diff
@@ -29,8 +29,8 @@
 | taidnguyen        |              | 14          |          |                     |            |                |           |                |          |               |
 | orionw            |              |             |          |                     |      14    |                |           |                |          |               |
 | mmhamdy           |              | 14          |          |                     |            |                |           |                |          |               |
-| manandey          |              | 12          |          |                     |            |                |           |                |          |               |
-| isaac-chung       |              | 36          |          |                     |            |                | 12        |                |          |               |
+| manandey          |              | 18          |          |                     |            |                |           |                |          |               |
```

Comment:
?

Reply:
Ah I should have clarified better: https://github.com/embeddings-benchmark/mteb/pull/437#issuecomment-2066055176

Reply:
If you update `manandey`'s points in #438 , I can remove it from this PR using the new system.

Reply:
Perfect!

Reply:
Added as new file to cover manandey's points.